web/settings.py

```diff
@@ -143,7 +145,14 @@
 # https://docs.djangoproject.com/en/3.2/howto/static-files/
 
 STATIC_URL = "/static/"
-STATIC_ROOT = env("STATIC_ROOT", default="/web/staticfiles")
+STATIC_ROOT = BASE_DIR / "staticfiles"
+
+STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
+
+STATICFILES_DIRS = [
+    BASE_DIR / "static",
+    "/var/www/static/",
+]
```

Comment:
What was the reason for changing this? Adding directories here shouldn't be necessary since the default finder `AppDirectoriesFinder` will already find the `static` directory into which Angular outputs its static files. In any case, there are no static files in the directories you listed anyway.

Reply:
I was actually planning on taking that out.